src/common/lightgbm.py

```diff
@@ -29,9 +29,12 @@ def callback(self, env: lightgbm.callback.CallbackEnv) -> None:
 
         # loop on all the evaluation results tuples
         for data_name, eval_name, result, _ in env.evaluation_result_list:
+            metric_key = f"{data_name}.{eval_name}"
+            if self.metrics_prefix:
+                metric_key = f"{self.metrics_prefix}.{metric_key}"
```

Comment:
Have we considered prefix vs postfix? Postfix might yield a more easily usable/scannable sorting in some scenarios.

Reply:
Discussed in the other pr #61 and filed as issue #63